src/scss/carousel.scss

```diff
@@ -0,0 +1,113 @@
+.mySlides {
+  display: none;
+}
+
+/* Slideshow container */
+.slideshow-container {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+}
+
+/* Next & previous buttons */
+.prev,
+.next {
+  background-color: #82827566;
```

Comment:
Invalid color value, hexadecimal color should be of 6 digits. And define the colors in variables files.

Reply:
Done